lib/CCreatureSet.h

```diff
@@ -109,6 +109,7 @@ class DLL_LINKAGE CStackInstance : public CBonusSystemNode, public CStackBasicDe
 	FactionID getFaction() const override;
 
 	virtual ui64 getPower() const;
+	virtual ui64 getCost() const;
```

Comment:
- use better name
- add comment to explain what it does
```suggestion
	/// Returns total market value of resources needed to recruit this unit
	virtual ui64 getMarketValue() const;
```